### PR TITLE
Main dts mtl tplg split 2 1

### PR DESCRIPTION
**Pull request by joechengxperi**
Based on main-dts-mtl-tplg-split-2 to modify the commits

### CODE REVIEW

File: tools/topology/topology2/cavs-mixin-mixout-eqiir-dts-ssp.conf [hunk 1]
```diff
@@ -0,0 +1,220 @@
+#
+# List of all DAIs
+#
+#SSP Index: 0, Direction: duplex
+#Object.Dai.SSP [
```
Comment (macchian) [inline tools/topology/topology2/cavs-mixin-mixout-eqiir-dts-ssp.conf]: duplicated in mix.conf. Remove SSP part.

File: tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake [hunk 2]
```diff
@@ -54,6 +54,13 @@ SPK_ID=6,SPEAKER_SSP_DAI_INDEX=0,HEADSET_CODEC_NAME=SSP2-Codec,SPEAKER_CODEC_NAM
 BT_NAME=SSP1-BT,BT_INDEX=1,BT_ID=7,BT_PCM_NAME=Bluetooth,INCLUDE_ECHO_REF=true,USE_CHAIN_DMA=true,\
 DEEPBUFFER_D0I3_COMPATIBLE=true"
 
+"cavs-rt5682\;sof-mtl-max98357a-rt5682-ssp2-ssp0-dts\;PLATFORM=mtl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,\
```
Comment (macchian) [inline tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake]: @joechengxperi , here we should apply dts playback on my WIP rt5650 topology2 PR here. https://github.com/macchian/sof/pull/16/commits/e63d230815c72bb8ba2f5ed0f97ffce705cc2ccf

Comment (joechengxperi) [reply tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake]: Ok, I'll remove it and patch it after your commit is merged.

File: tools/topology/topology2/cavs-mixin-mixout-eqiir-dts-ssp.conf (hunk 1 above)
Comment (joechengxperi) [reply tools/topology/topology2/cavs-mixin-mixout-eqiir-dts-ssp.conf]: @macchian The duplicated parts are removed now.

File: tools/topology/topology2/cavs-rt5682.conf [hunk 3]
```diff
@@ -105,6 +106,7 @@ Define {
 	GOOGLE_AEC_PIPELINE_PRIORITY	2
 	DMIC_PIPELINE_PRIORITY		1
 	ECHO_REF_PIPELINE_PRIORITY	0
+	SSP_CONFIG                      "mix"
```
Comment (macchian) [inline tools/topology/topology2/cavs-rt5682.conf]: maybe consider name as "PLAYBACK_PIPELINE_SRC" 
"volume"
or
"eqiir"

Comment (joechengxperi) [reply tools/topology/topology2/cavs-rt5682.conf]: @macchian Done. Please check the latest commit.

**Pull request closed without merging**